kotlin/vci-client/src/main/java/io/mosip/vciclient/authorizationCodeFlow/interactiveAuthorization/presentationDuringIssuance/PresentationDuringIssuanceAuthorizationMethodService.kt

```
@@ -0,0 +1,195 @@
+        } catch (error: Exception) {
+            logger.warning("Error during presentation handling: ${error.message}")
+            vpResponse = openId4vp.constructErrorInfo(error)
+        }
+
+        return try {
+            sendOVPAuthorizationResponseToIssuer(
+                iar = requestData.iar,
+                authSession = requestData.authSession,
+                vpResponse = vpResponse
+            )
+        } catch (ex: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)